Makefile

```diff
@@ -791,7 +805,9 @@ edpm_deploy_prep: edpm_deploy_cleanup ## prepares the CR to install the data pla
 	cp ${DATAPLANE_SERVICE_NOVA_CR} ${DEPLOY_DIR}
 	cp ${DATAPLANE_NODESET_CR} ${DEPLOY_DIR}
 	cp ${DATAPLANE_DEPLOYMENT_CR} ${DEPLOY_DIR}
+ifneq ($(DATAPLANE_EXTRA_NOVA_CONFIG_FILE),)
 	cp ${DATAPLANE_EXTRA_NOVA_CONFIG_FILE} ${EDPM_EXTRA_NOVA_CONFIG_FILE}
+endif
```

Comment:
I think this is an unnecessary change if you keep the default for the env variable

Reply:
ack. followed existing code format. will update it.